### PR TITLE
Use Cognite 3D team tslint config (same as viewer uses)

### PR DESCRIPTION
**Pull request by nsunderland-cognite**
https://www.npmjs.com/package/@cognite/tslint-config-cognite-3d

### CODE REVIEW

File: src/__tests__/InstancedMeshGroup.test.ts [hunk 1]
```diff
@@ -21,40 +21,15 @@ describe('MeshGroup', () => {
...
     const nodeIds = [11, 12, 13, 14];
     const treeIndices = [10, 20, 30, 40];
-    const transformMatrices = [
-      new THREE.Matrix4(),
-      new THREE.Matrix4(),
-      new THREE.Matrix4(),
-      new THREE.Matrix4(),
-    ];
-
-    transformMatrices[0].set(
```
Comment (nsunderland-cognite) [inline src/__tests__/InstancedMeshGroup.test.ts]: I should probably disable prettier for these matrices

File: src/geometry/MergedMeshGroup.ts [hunk 2]
```diff
@@ -70,16 +76,11 @@ export class MergedMeshMappings {
...
-      n11, n12, n13, n14,
-      n21, n22, n23, n24,
-      n31, n32, n33, n34,
-      0,   0,   0,   1,
-    );
+    const [n11, n21, n31] = columns[0];
+    const [n12, n22, n32] = columns[1];
+    const [n13, n23, n33] = columns[2];
+    const [n14, n24, n34] = columns[3];
+    target.set(n11, n12, n13, n14, n21, n22, n23, n24, n31, n32, n33, n34, 0, 0, 0, 1);
```
Comment (nsunderland-cognite) [inline src/geometry/MergedMeshGroup.ts]: Disable prettier here too

File: src/geometry/QuadGroup.ts [hunk 3]
```diff
@@ -62,28 +62,18 @@ export default class QuadGroup extends PrimitiveGroup {
     side1.normalize();
     side2.normalize();
 
-    basis.set(
-      side2.x, side1.x, normal.x, 0,
-      side2.y, side1.y, normal.y, 0,
-      side2.z, side1.z, normal.z, 0,
-            0,       0,        0, 1,
-    );
+    basis.set(side2.x, side1.x, normal.x, 0, side2.y, side1.y, normal.y, 0, side2.z, side1.z, normal.z, 0, 0, 0, 0, 1);
```
Comment (nsunderland-cognite) [inline src/geometry/QuadGroup.ts]: Disable prettier here

File: src/geometry/QuadGroup.ts [hunk 4]
```diff
@@ -62,28 +62,18 @@ export default class QuadGroup extends PrimitiveGroup {
...
     outputMatrix.premultiply(basis);
 
     center.addVectors(globalVertex1, globalVertex2).multiplyScalar(0.5);
-    basis.set(
-      1, 0, 0, center.x,
-      0, 1, 0, center.y,
-      0, 0, 1, center.z,
-      0, 0, 0, 1,
-    );
+    basis.set(1, 0, 0, center.x, 0, 1, 0, center.y, 0, 0, 1, center.z, 0, 0, 0, 1);
```
Comment (nsunderland-cognite) [inline src/geometry/QuadGroup.ts]: And here

**Review by anderscognite — APPROVED** [review]
Crazy PR!

File: src/parsers/i3d/PropertyLoader.ts [hunk 5]
```diff
@@ -37,69 +37,71 @@ export default class PropertyLoader {
...
+    // tslint:disable:prettier
+    treeIndex: indices => { this.treeIndex =               indices.nextValue(); },
+    color: indices => {
       const index = indices.nextValue();
       if (index === 0) {
         this.color                              = DEFAULT_COLOR;
       } else {
         this.color                              = this.values.color![index - 1];
       }},
-    'center': (indices: FibonacciDecoder) => {
```
Comment (anderscognite) [inline src/parsers/i3d/PropertyLoader.ts:49]: heh

**Pull request merged**